shell/platform/darwin/macos/framework/Source/FlutterGLCompositor.mm

```diff
@@ -19,13 +19,16 @@
 
 namespace flutter {
 
-FlutterGLCompositor::FlutterGLCompositor(FlutterViewController* view_controller,
+FlutterGLCompositor::FlutterGLCompositor(FlutterViewProvider* view_provider,
                                          NSOpenGLContext* opengl_context)
-    : FlutterCompositor(view_controller), open_gl_context_(opengl_context) {}
+    : FlutterCompositor(view_provider), open_gl_context_(opengl_context) {}
 
 bool FlutterGLCompositor::CreateBackingStore(const FlutterBackingStoreConfig* config,
                                              FlutterBackingStore* backing_store_out) {
-  if (!view_controller_) {
+  // Always gets the first view, #0. After Flutter supports multi-view, it
+  // should get the view ID from somewhere.
+  FlutterView* view = GetView(0);
```

Comment:
Shouldn't this be using the constant?

Reply:
@dkwingsmt I think you meant to use: kFlutterDefaultViewId here